mozilla/bedrock
# Bug 1010596 fxos devices zte openc

Ready for review:
- Adds eBay CTA links to the `firefox/os/devices` page for Germany, Russia, United States and United Kingdom.
- Because these phones are aimed more at developers, we only want to show them on the `os/devices` page, and not on `firefox/os`. To do this we've added a `developer_only` property to the partner data that both pages share. The `firefox/os` page will not show a link for a given provider if this property is present.

Apologies for the massive diff in `firefox-os.js`, 90% of this is just indentation changes. The only real change is a minor refactor to [this function](https://github.com/mozilla/bedrock/compare/bug-1010596-fxos-devices-zte-openc?expand=1#diff-829159f527bc87788df23a0863b651f3R19).

How to test this PR:
- Visit `firefox/os/devices` and select each of the countries listed above from the drop down menu. Verify that the eBay link is shown for each when clicking the blue CTA button.
- Visit `firefox/os/` and set `COUNTRY_CODE` = `'de'` in `firefox-os.js` on [this line](https://github.com/mozilla/bedrock/pull/2024/files#diff-829159f527bc87788df23a0863b651f3R117). You should then see a green CTA link, but it should only list one provider (not the eBay link). If `COUNTRY_CODE` is set to `us` or `gb` or `ru`, the green CTA should not show.
